_content/essays/worlds-and-their-cessation_geoff

```diff
@@ -0,0 +1,23 @@
+---
+title: "Worlds & Their Cessation: The Buddha's Strategic View of the Cosmos"
+authors:
+  - geoff
+subcat: 
+external_url: "https://www.dhammatalks.org/Archive/Writings/CrossIndexed/Uncollected/MiscEssays/Worlds.pdf"
+source_url: "https://www.dhammatalks.org/books/uncollected/Worlds.html"
+drive_links:
+  - "https://drive.google.com/drive/folders/10eW0WUeTrF7_GLU_efLIQlrteSbxz474"
+formats: [pdf]
+course: right view
+tags:
+  - cosmology
+  - world
```

Comment:
I see where you're coming from with `world` 🤔 but it still feels more philosophical than anthropological. If you think it deserves a place on the secular side of the library, maybe `ideology`? Or maybe even more specifically, `materialism`?